lib/ecto/adapters/postgres/connection.ex

```diff
@@ -381,6 +381,10 @@ if Code.ensure_loaded?(Postgrex.Connection) do
     alias Ecto.Migration.Index
     alias Ecto.Migration.Reference
 
+    def supports_ddl_transaction? do
```

Comment:
Should we delegate this to the connection or just implement it directly in Ecto.Adapters.Postgres? I think there is no need for delegation in this case as long as we don't implement it in `__using__` below?